### PR TITLE
Update ServiceController version for netstandard support

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This is required for https://github.com/aspnet/Hosting/pull/1189

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -366,7 +366,7 @@
     <ExternalDependency Include="System.Security.Cryptography.Xml" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="System.Security.Permissions" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="System.Security.Principal.Windows" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.ServiceProcess.ServiceController" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.ServiceProcess.ServiceController" Version="4.5.0-preview1-25914-04" Source="$(DefaultNuGetFeed)" />
```
Comment (pranavkm) [inline build/dependencies.props]: This comes from dotnet-core feed (not `DefaultNuGetFeed`). Also needs to be mirrored.

Comment (pranavkm) [reply build/dependencies.props]: While we're at this, should we do a pass through repos and bump them up to use the latest CoreFx packages too? We updated the SDK \ runtime, presumably packages should be next. @Eilon \ @muratg thoughts?

Comment (JunTaoLuo) [reply build/dependencies.props]: This one is actually on nuget: https://www.nuget.org/packages/System.ServiceProcess.ServiceController/4.5.0-preview1-25914-04

Comment (muratg) [reply build/dependencies.props]: @pranavkm yes, I think we should bump the packages as well.

Comment (natemcmaster) [reply build/dependencies.props]: @pranavkm chat with @ryanbrandenburg about #577. We need to make sure we make these version updates work well with Orchestrated Build.

Comment (pranavkm) [reply build/dependencies.props]: Sure. But I was just hoping to do a one time update for these versions. Presumably the orchestrated build versions would be the latest (4.5.0-*) versions and not the shipped packages.

**Pull request merged**